docs/network-flow-visibility.md

```diff
@@ -37,24 +47,24 @@ Services, Pods etc., and thereby provides opportunities to enhance the performan
 and security aspects of Pod workloads.
 
 For visualizing the network flows, Antrea monitors the flows in Linux conntrack
-module. These flows are converted to flow records and are sent to the configured
-flow controller. High-level design is given below:
+module. These flows are converted to flow records, flow records are post-processed
+and are sent to the configured external flow collector. High-level design is given below:
 
-![Flow Exporter Design](assets/flow_exporter.svg)
+![Antrea Flow Visibility Design](assets/flow_visibility.svg)
 
-## Flow Exporter feature
+## Flow Exporter
 
 In Antrea, the basic building block for the Network Flow Visibility is the **Flow
-Exporter feature**. Flow Exporter operates within Antrea Agent; it builds and maintains
+Exporter**. Flow Exporter operates within Antrea Agent; it builds and maintains
 a connection store by polling and dumping flows from conntrack module periodically.
-Connections from the connection store are exported to a flow collector using the
-IPFIX protocol, and for this purpose we use the [go-ipfix](https://github.com/vmware/go-ipfix) library.
+Connections from the connection store are exported to the [Flow Aggregator
+service](#flow-aggregator) using the IPFIX protocol, and for this purpose we use
+the IPFIX exporter process from [go-ipfix](https://github.com/vmware/go-ipfix) library.
 
 ### Configuration
 
 To enable the Flow Exporter feature at the Antrea Agent, the following config
-parameters have to be set in the Antrea Agent ConfigMap as shown below. We provide
-some examples for the parameter values in the following snippet.
+parameters have to be set in the Antrea Agent ConfigMap as shown below.
```

Comment:
nit: a bit redundant considering "the following" at the beginning of the sentence